src/content/glossary/index.md

```diff
@@ -371,9 +374,9 @@ A label given to some [EIPs](#eip) that attempt to define a specific standard of
 
 ### Ethash {#ethash}
 
-A [proof-of-work](#pow) algorithm for Ethereum 1.0.
+A [proof-of-work](#pow) algorithm that was used on Ethereum before it transitioned to [proof-of-stake](#pos).
 
-[Read more at eth.wiki](https://eth.wiki/en/concepts/ethash/ethash)
+[Read more](/developers/docs/consensus-mechanisms/pow/mining-algorithms/ethash)
```

Comment:
Thoughts on removing this entirely? Doesn't have to be now but I don't think it's necessary to keep it in the glossary forever...

Reply:
idk i think it's good to keep for now - people will still hear about it in various places and it's good to have a simple explanation and clarify that it's no longer used.